src/hypergraph/runners/_shared/validation.py

```
@@ -0,0 +1,202 @@
+    )
+
+    raise MissingInputError(
+        missing=missing,
+        provided=list(provided),
+        message=message,
+    )
+
+
+def _get_suggestions(
+    missing: list[str],
+    all_inputs: set[str],
```

Ruff (0.14.13)

Unused function argument: `all_inputs`

(ARG001)